backend/src/services/auth/authService.ts

```diff
@@ -482,12 +482,13 @@ class AuthService {
         )
       }
       // If there was no provider, we can link it to the provider
-      if (user && (user.provider === undefined || user.provider === null)) {
+      if (user && (user.provider === undefined || user.provider === null || user.emailVerified)) {
```

Comment:
Wasnt here -user.emailverified

Reply:
It's this change https://github.com/CrowdDotDev/crowd.dev/commit/d77fae237026329bf9dba7431e5412965da487ce#diff-9a960a7047333c42ea59b7d48d1a0b9cedd8fc8c3562bafee6e7273db7c19deeR485